oai_agents/common/population.py

```diff
@@ -13,21 +13,30 @@ def train_agent_with_checkpoints(args, total_training_timesteps, ck_rate, seed,
         Returns ckeckpoints_list
         either serialized or not based on serialize flag
     '''
-
     name = f'SP_hd{h_dim}_seed{seed}'
 
+    agent_ckpt = None
+    start_step = 0
+    if args.resume:
+        agent_ckpt_info, env_info = RLAgentTrainer.load_agents(args, name=name, tag='best')
```

Comment:
tag should not be 'best', it should be the last ck_ that's available in the directory

Reply:
Yeah i think id need the remove aamas branch merged for that. Just needed a checkpoint that was always there to test. will change before merge